src/current/_includes/v23.1/sidebar-data/reference.json

```diff
@@ -1527,6 +362,93 @@
           }
         ]
       },
+      {
+        "title": "Metrics",
+        "urls": [
+          "/${VERSION}/metrics.html"
+        ]
+      },
+      {
+        "title": "Logs",
+        "items": [
+          {
+            "title": "Logging Levels and Channels",
+            "urls": [
+              "/${VERSION}/logging.html"
+            ]
+          },
+          {
+            "title": "Log Formats",
+            "urls": [
+              "/${VERSION}/log-formats.html"
+            ]
+          },
+          {
+            "title": "Notable Event Types",
+            "urls": [
+              "/${VERSION}/eventlog.html"
+            ]
+          }
+        ]
+      },
+      {
+        "title": "Cluster Settings",
+        "urls": [
+          "/${VERSION}/cluster-settings.html"
+        ]
+      },
+      {
+        "title": "Diagnostics Reporting",
+        "urls": [
+          "/${VERSION}/diagnostics-reporting.html"
+        ]
+      },
+      {
+        "title": "Policies",
+        "items": [
+          {
+            "title": "CockroachDB Feature Availability",
+            "urls": [
+              "/${VERSION}/cockroachdb-feature-availability.html"
+            ]
+          },
+          {
+            "title": "Unsupported Features in CockroachDB Serverless",
+            "urls": [
+              "/cockroachcloud/serverless-unsupported-features.html"
+            ]
+          },
+          {
+            "title": "API Support Policy",
+            "urls": [
+              "/${VERSION}/api-support-policy.html"
+            ]
+          }
+        ]
+      },
```

Comment:
I think Release Support Policy still needs to be below this item:

<img width="431" alt="image" src="https://github.com/cockroachdb/docs/assets/8740013/015d8c35-846e-4951-82cc-7cd0bbdcd17e">

Reply:
Release Support Policy is staying under Releases until the last phase of updates to account for the releases sidebar being unversioned (so dupes of the link might appear)